src/modules/Transcript/index.tsx

```diff
@@ -770,8 +778,13 @@ function useAudioSync(
         // If we aren't listening at all and the call is live, we also want to
         // default to the partials
         if (listeningAtLiveEdge || liveAndNotListening) {
-            if (partial.text) setCurrentParagraph('partial');
-            else {
+            if (partial.text) {
+                setCurrentParagraph('partial');
+                // Not playing audio
+                if (!isListening) {
+                    audioPlayer.seekToEnd();
```

Comment:
since the partial is being selected - I'm seeking the audio if its not currently playing